conans/client/conf/__init__.py

```diff
@@ -684,3 +688,22 @@ def get_log_level_by_name(level_name):
             "notset": logging.NOTSET
         }
         return levels.get(str(level_name).lower())
+
+    @property
+    def config_install_interval(self):
```

Comment:
@uilianries I have moved the validation of the interval here. It doesn't make sense that an incorrect value escape from this, and has to be validated in different places of the codebase, possibly with duplication of code.